src/main/java/com/example/medicare_call/global/enums/NotificationType.java

```diff
@@ -0,0 +1,7 @@
+package com.example.medicare_call.global.enums;
+
+public enum NotificationType {
+    CALL_COMPLETED,     // 케어콜 완료
+    HEALTH_ALERT,       // 건강 이상 징후
+    MISSED_CALL         // 부재중 알림
```

Comment:
```suggestion
    CALL_STATUS_NO_ANSWER         // 부재중 알림
```

callStatus에서 쓰는 거랑 맞추는게 나중에 찾기 좋을 것 같아요.
지금보니 callStatus필드도 enum 타입으로 마이그레이션이 필요하겠네요..